client_end_user/shared/src/commonMain/kotlin/resources/DrawableResources.kt

```diff
@@ -0,0 +1,31 @@
+package resources
+
+data class DrawableResources(
+    val bpIcon: String = "ic_beep_beep_icon.xml",
```

Comment:
I think you need to delete all this resources just set one for test

Reply:
Done